addons/onboarding/controllers/onboarding.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.http import request
+
+
+class OnboardingController(http.Controller):
+    @http.route(['/onboarding/<int:onboarding_id>',
+                 '/onboarding/<string:onboarding_alias>'], auth='user', type='json')
+    def get_onboarding_data(self, onboarding_id=None, onboarding_alias=None):
+        if not request.env['onboarding.onboarding'].check_access_rights('read', raise_exception=False):
```

Comment:
Pro of using is_admin is that it is cached (normally), while here you could have some extra queries. @awa-odoo seems this request is from you, not sure :)

Reply:
I was not a fan of hardcoded "is_admin" right before a search.
Idea was to let the ACL do the job, which seemed cleaner, and would also work if we ever want to use the onboarding for other users.

But if is_admin is more efficient, then we can go back to that.

(As discussed with Florian, then I would rather have that "is_admin" check at the JS level to avoid unnecessary rpc).